Azure-Samples/Microchip-PIC-IoT-Wx
# IoT Plug and Play GA support

- Add a device model and code for the model
- Re-factored LED code
- Refactored various code to keep the binary size < 0xAF00 bytes
- Set indent for debug print
- Remove unnecessary traces
- Consistent coding style (Tab vs space etc)
- Add CLI to configure DPS ID Scope
- Add CLI to control LED over UART
- Add button press as event telemetry

## Purpose
Certify the device for IoT Plug and Play and publish to the device catalog.

## Does this introduce a breaking change?
[ ] Yes
[x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

[x ] Bugfix
[x] Feature
[x ] Code style update (formatting, local variables)
[x ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Flash the FW and run the IoT Plug and Play certification test.

* Test the code
- Use CLI to control LED to make sure properties are reported to IoT Hub
- Provision device using DPS
- Send Commands from Cloud
- Update Writable property to control LED and telemetry interval

## Other Information
